src/main/java/org/sonarqube/gradle/SonarUtils.java

```diff
@@ -190,9 +190,17 @@ static List<File> exists(Iterable<File> files) {
   }
 
   static void appendProps(Map<String, Object> properties, String key, Iterable<?> valuesToAppend) {
-    properties.putIfAbsent(key, new LinkedHashSet<String>());
-    StreamSupport.stream(valuesToAppend.spliterator(), false)
-      .forEach(((Collection<Object>) properties.get(key))::add);
+    Set<Object> newList = new LinkedHashSet<>();
```

Comment:
Nitpick: because this is not a list, can we rename the variable to `newProps`, `combinedProperties` or something else?